### PR TITLE
feat(tooltip): review component

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?
Review of the tooltip component.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

File: packages/admin-ui/src/tooltip/tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  const children: FunctionComponentElement<unknown> = _children ?? (
+    <TooltipTrigger bleedX={bleedX} bleedY={bleedY} />
+  )
+
+  return (
+    <>
+      <TooltipAnchor
+        state={state}
+        {...(children.props as any)}
```
Comment (anitavincent) [inline packages/admin-ui/src/tooltip/tooltip.tsx:47]: Why was the any necessary?

Comment (anitavincent) [reply packages/admin-ui/src/tooltip/tooltip.tsx:47]: Could _children get some more specific type that wouldn't cause a conflict?

**Review by anitavincent — APPROVED** [review]

File: packages/admin-ui-docs/content/tooltip/code.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| bleedY    | `boolean`                                                                           | Vertical bleed                                        | 🚫       | `false` |
+
+</Section>
+
+<Section direction="row">
+
+## Accessibility
+
+<OrderedList>
+  <OrderedListItem>`Tooltip` has `role="tooltip"`.</OrderedListItem>
```
Comment (anitavincent) [inline packages/admin-ui-docs/content/tooltip/code.mdx:39]: These ` (plicks or crases) are buggy on the live example. Don't know why

Comment (Lenderson) [reply packages/admin-ui-docs/content/tooltip/code.mdx:39]: I think it is the way OrderedListItem is implemented. We will need to update it to be able to accept the markdown language., probably similar to what we did to `<Section/>` component

File: packages/admin-ui-docs/content/tooltip/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+<Tooltip text="Tooltip text" />
+```
+
+</Section>
+
+<Section>
+
+### Target
+
+You can change the tooltip target through children.
```
Comment (Lenderson) [inline packages/admin-ui-docs/content/tooltip/overview.mdx:26]: You will need to add the boolean prop `hasExplanation` to the <Section/> to let the text with the correct spacing between the title and the live preview code

File: packages/admin-ui/src/tooltip/tooltip.tsx (hunk 1 above)
Comment (matheusps) [reply packages/admin-ui/src/tooltip/tooltip.tsx:47]: Changed to `tooltipAnchorChildren`. The any is there for a typescript edge case.

File: packages/admin-ui-docs/content/tooltip/code.mdx (hunk 2 above)
Comment (matheusps) [reply packages/admin-ui-docs/content/tooltip/code.mdx:39]: So, I will leave it here

**Pull request merged**